src/Infrastructure/Service/Filesystem.php

```diff
@@ -5,7 +5,7 @@
  * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
  */
 
-namespace SprykerSdk\Sdk\Core\Application\Service;
+namespace SprykerSdk\Sdk\Infrastructure\Service;
 
 use Symfony\Component\Filesystem\Filesystem as SymfonyFilesystem;
```

Comment:
brancho blocked changes because uses this path in Init class

Reply:
there're 2 commands that use ROOT_DIR env in the configuration. for now it moved to entry point file.